internal/constants.go

```diff
@@ -20,7 +20,7 @@ const (
 	BootTimeoutDefault   = 30000
 	ClientMonitorDefault = 15000
 	ConfigFileName       = "configuration.toml"
-	ConfigRegistryStem   = "edgex/appfunctions/1.0/"
+	ConfigRegistryStem   = "edgex/appservices/1.0/"
```

Comment:
Be advised this isn't necessarily a backward compatible change b/c if someone has customized their appfunctions branch in Consul and expects the new service to stand up and use that same branch with its overrides, that won't happen. A new branch will be created.

Reply:
@tsconn23 , Yep. At a minimum we need to document the change for Fuji. Another topic to discuss in tomorrow's App Services WG meeting. :-)